### DISCUSSION TITLE
SO stops collecting events

### DISCUSSION THREAD

**Discussion by marianoka** [Security-Onion-Solutions/securityonion#6129] [General]
Hi So gurus,
I have an issues I can´t get into it. The thing is: SO is not showing events. I have checkd at grid item and I see a faul alert.
![imagen](https://user-images.githubusercontent.com/9213379/140642738-40f1c180-321d-4c75-bafa-e8865ab54fd6.png)
Any Idea what log should I see to understand and fix the problem?
Thanks
Mariano

**Comment by marianoka** [comment]
what I'm seeing is after doing a curl to elk, that no index is being created.

**Reply by Acewiza** [reply]
So-status-all?  Any background?  Last time mine suddenly stopped alerting for no "apparent" reason, was right after I borked the global.sls attempting BPF configs.
